drivers/rn2xx3/rn2xx3_getset.c

```diff
@@ -141,14 +127,11 @@ static void _get_array_value(rn2xx3_t *dev, const char* command,
 static void _set_array_value(rn2xx3_t *dev, const char* command,
                              uint8_t *value, uint8_t value_len)
 {
-    dev->cmd_buf[0] = 0;
     snprintf(dev->cmd_buf, sizeof(dev->cmd_buf) - 1, "%s %s %s ",
              mac, set, command);
```

Comment:
Missing the cmd_buf[p] = 0 after the snprintf

Reply:
Fixed